tests/console/libjpeg_turbo.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: libjpeg_turbo
+# Summary: Check libjpeg_turbo version
+# - Test should test on sle15sp6, see https://jira.suse.com/browse/PED-4889
+# - Check libjpeg-turbo version is higher than 2.1.1
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use version_utils;
+use Mojo::Util 'trim';
+use utils "zypper_call";
+use registration qw(add_suseconnect_product is_phub_ready);
+
+sub run {
+    select_serial_terminal;
+
+    # Package 'libjpeg-turbo' requires PackageHub is available
+    add_suseconnect_product('PackageHub') unless is_phub_ready();
+
+    # Install libjpeg-turbo package
+    zypper_call("install libjpeg-turbo");
+    my $version = script_output("zypper info libjpeg-turbo | grep '^Version'");
+    record_info("libjpeg-turbo version", $version);
+    my @install_version = split(/:/, $version);
+    die("libjpeg-turbo version is not update") if (package_version_cmp(trim($install_version[1]), '2.1.1') <= 0);
+}
+
+sub post_fail_hook {
+    select_console 'log-console';
+    assert_script_run 'save_y2logs /tmp/libjpeg-turbo_version_check_y2logs.tar.bz2';
+    upload_logs '/tmp/libjpeg-turbo_version_check_y2logs.tar.bz2';
```

Comment:
Need to call the baseclass method here, otherwise logs will be missing:

```
sub post_fail_hook {
    my $self = shift;
    ...
    $self->SUPER::post_fail_hook;
}
```